### PR TITLE
✨Add Total Score gameboard component

### PR DESCRIPTION
**Pull request by fre-ben**
- Add Total Score component
- Add to storybook

Deployment: https://sisu-pipelin-totalscore-b0njmv.herokuapp.com/storybook/index.html?path=/story/common-gameboard--totalscore
Part of issue #6

### CODE REVIEW

**Review by mhstrkmp — COMMENTED** [review]
lgtm 🚀

File: components/gameboard/GameBoard.stories.tsx [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 import DiscardPile from "./DiscardPile";
 import DrawPile from "./DrawPile";
+import TotalScore from "./TotalScore";
 
 export default {
   title: "Common/Gameboard",
 } as Meta;
 
 export const drawpile = () => <DrawPile onClick={() => alert("test")} />;
 export const discardile = () => <DiscardPile onClick={() => alert("test")} />;
```
Comment (mewiemers) [inline components/gameboard/GameBoard.stories.tsx]: maybe one of them is a typo or not?

```suggestion
export const discardpile = () => <DiscardPile onClick={() => alert("test")} />;
```

**Review by mewiemers — COMMENTED** [review]
GJ! 🦡

**Review by RonjaMeinecke — APPROVED** [review]
🦾GJ

File: components/gameboard/TotalScore.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import styles from "./Score.module.css";
+
+function TotalScore() {
+  // Example data, will be removed later by Props handed down to TotalScore
+  const players = [
+    { name: "Frederik", score: 20 },
+    { name: "Kalle", score: 10 },
+    { name: "Eva", score: 30 },
+    { name: "Louis", score: 60 },
+    { name: "Frederik", score: 20 },
```
Comment (HorstFrank) [inline components/gameboard/TotalScore.tsx:10]:
```suggestion
    { name: "Boris", score: 20 },
```

**Review by HorstFrank — APPROVED** [review]
GJ

File: components/gameboard/GameBoard.stories.tsx (hunk 1 above)
Comment (fre-ben) [reply components/gameboard/GameBoard.stories.tsx]: yup, thanks!

File: components/gameboard/TotalScore.tsx (hunk 2 above)
Comment (fre-ben) [reply components/gameboard/TotalScore.tsx:10]: Such an obvious mistake

**Pull request merged**